troubadix/helper/linguistic_exception_handler.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2021-2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import re
+from abc import ABC, abstractmethod
+from typing import Iterable, List, Tuple, Union
+
+
+class LinguisticCheck(ABC):
+    @abstractmethod
+    def execute(self, file, content):
```

Comment:
I find `file` a little bit confusing, because it's not checked using `==`, but `in`. Can we clarify this somewhere to have a clean interface? But honestly, I have no idea how to do so in a non-ugly way, so feel free to resolve.

Reply:
We could call it `filename` (but it can be about more than the name) or `path` (but it is not a `pathlib.Path`) and it mirrors (and there by, preserves) the previous handling.

Reply:
Maybe `file_path` would fit better with the `in` handling ?

Reply:
Sounds good, `file_path` seems to get us as close as possible :)